src/customHooks/useLeaderBoard/useLeaderBoard.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "@tanstack/react-query";
+import { getLeaderBoardFromLocalStorage } from "../../api/localStorageApi";
+import { errorMessage } from "./strings";
+
+
+export function useLeaderBoard(gameId: number) { 
+    const { data } = useQuery([gameId.toString], getLeaderBoardFromLocalStorage, {

```

Comment:
maybe memory addresses are very random, but man!
```suggestion
    const { data } = useQuery([gameId.toString()], getLeaderBoardFromLocalStorage, {

```